src/osmet/mod.rs

```diff
@@ -150,6 +151,23 @@ pub fn osmet_unpack(config: &OsmetUnpackConfig) -> Result<()> {
     Ok(())
 }
 
+pub fn find_matching_osmet_in_dir(sector_size: NonZeroU32, osmet_dir: &Path) -> Result<PathBuf> {
```

Comment:
Per https://github.com/coreos/coreos-installer/pull/187#discussion_r402681315 we need to care about target CPU arch also.

I'd put `osmet_dir` first.  We'll probably have more matching parameters over time.